changelog/README.md

```diff
@@ -8,6 +8,10 @@ For more details, see [Environments](../guidelines/environments.md).
 
 # Changelog
 
+## 22th August 2022
+
+* Added new type `CustomerAdded` and `CustomerUpdated` of webhook [Event](../webhooks/wh-general.md#Supported events).
+
```

Comment:
Note when adding internal links to headings like "Supported events", you need to change it to lower case and replace spaces with '-'. So in this case it becomes #supported-events.

Actually, I think we could word this more simply by saying "Added new webhooks" or "Added new webhook events". So I suggest the following:

* Added new webhook events `CustomerAdded` and `CustomerUpdated` in [General Webhooks](../webhooks/wh-general.md).

Reply:
I've changed it to your suggestions.